RxSwiftWorkshop/Shared/FlowController.swift

```diff
@@ -0,0 +1,31 @@
+//
+// Created by Krzysztof Siejkowski on 03/05/2017.
+// Copyright (c) 2017 Mobile Academy. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+final class FlowController {
+
+    func setupRootViewController(for window: UIWindow) {
+        window.rootViewController = createFlightsViewController()
+    }
+
+    private func createFlightsViewController() -> UIViewController {
+        let jsonClient = JSONAPIClient()
+        let schipolService = SchipolService(client: jsonClient)
+        let iataService = IATAService(client: jsonClient)
+        let flightViewModel = FlightsViewModel(schipolCallable: schipolService, iataCallable: iataService)
+        let flightsViewController = FlightsViewController(viewModel: flightViewModel, showDetailsViewController: showMapViewController)
+        return UINavigationController(rootViewController: flightsViewController)
+    }
+
+    func showMapViewController(flight: Flight, from fromViewController: UIViewController) {
```

Comment:
`showMapViewController(for flight: Flight, from fromViewController: UIViewController)`?